### DISCUSSION TITLE
Build subversion with `--with-apxs` so svn+http can be enabled

### DISCUSSION THREAD

**Discussion by kklem0** [Homebrew/discussions#4637] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
no
```

### Output of `brew doctor`

```shell
no
```

### Description of issue

Now `mod_dav_svn.so` and `mod_authz_svn.so` are missing and it's because subversion builds with `--with-apxs=no`.
Changing it to `--with-apxs=/opt/homebrew/bin/apxs` would then generate the required bits for enabling svn+http which is important for svn.

**Comment by kklem0** [comment]
1) `brew install httpd`
2) `brew edit subversion`
3) `--with-apxs=no` -> `--with-apxs=/opt/homebrew/bin/apxs`
4) `export HOMEBREW_NO_INSTALL_FROM_API=1`
5) `brew install subversion --verbose --build-from-source`

**Comment by p-linnane** [comment]
Please open a PR for this.
